.readthedocs.yaml

```diff
@@ -0,0 +1,24 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
+  jobs:
+    pre_build:
+      - pip install .
```

Comment:
not poetry install?

Reply:
The default command rtfd used is “python -m sphinx_xxx xxx” and that will raise ModuleNotFound error with "import tianshou" in docs/conf.py, since poetry has another set of self-defined env var

Reply:
I don't want to change the default command they used to build since it might change, so that's why I use pip instead of poe here